### PR TITLE
STJ migration

### LINKED ISSUE #121: Migrate to System.Text.Json
Remove `Newtonsoft.Json` dependency in favor of `System.Text.Json`.

### PR DESCRIPTION
**Pull request by Seb-stian**
Closes #121

It's finally here folks! Thank you @Tides

### CODE REVIEW

File: Obsidian/Utilities/Config.cs [hunk 1]
```diff
@@ -1,63 +1,44 @@
...
         public bool MulitplayerDebugMode { get; set; }
 
-        [JsonProperty("header")]
         public string Header { get; set; } = "§dObsidian > All other servers";
 
-        [JsonProperty("footer")]
         public string Footer { get; set; } = "§5tiddies §l§d( §c. §d)( §c. §d)";
 
-        [JsonProperty("baah", NullValueHandling = NullValueHandling.Ignore)]
         public bool? Baah { get; set; }
```
Comment (Naamloos) [inline Obsidian/Utilities/Config.cs:34]: make sure this is not serialized when not set. it's a secret:tm:

**Review by Naamloos — APPROVED** [review]
based

File: Obsidian/Utilities/Config.cs (hunk 1 above)
Comment (Tides) [reply Obsidian/Utilities/Config.cs:34]: I set the JsonSerializerOptions to not include null/default objects ^ ^

File: Obsidian/Utilities/Converters/IngredientsConverter.cs [hunk 2]
```diff
@@ -1,18 +1,6 @@
...
-using System.Collections.Generic;
-using System.Collections.ObjectModel;
-using System.Linq;
-using static Obsidian.Utilities.Registry.Registry;
-
-namespace Obsidian.Utilities.Converters
+﻿namespace Obsidian.Utilities.Converters
 {
-    public class IngredientsConverter : JsonConverter
+    /*public class IngredientsConverter : JsonConverter
```
Comment (Seb-stian) [inline Obsidian/Utilities/Converters/IngredientsConverter.cs]: Is this up for deletion?

**Review by Seb-stian — COMMENTED** [review]
Very good

File: Obsidian/Utilities/Converters/IngredientsConverter.cs (hunk 2 above)
Comment (Naamloos) [reply Obsidian/Utilities/Converters/IngredientsConverter.cs]: could be a temporary comment that he forgot to kill

Comment (Tides) [reply Obsidian/Utilities/Converters/IngredientsConverter.cs]: Yeah I forgot to delete 😅

**Pull request merged**